portfolio/.gcloudignore

```diff
@@ -0,0 +1,17 @@
+# This file specifies files that are *not* uploaded to Google Cloud Platform
+# using gcloud. It follows the same syntax as .gitignore, with the addition of
+# "#!include" directives (which insert the entries of the given .gitignore-style
+# file at that point).
+#
+# For more information, run:
+#   $ gcloud topic gcloudignore
+#
+.gcloudignore
+# If you would like to upload your .git directory, .gitignore file or files
+# from your .gitignore file, remove the corresponding line
+# below:
+.git
+.gitignore
+
+# Target directory for maven builds
+target/
```

Comment:
minor: add a new line at the end of the file, this is common practice for files uploaded to git. It doesn't do anything except make sure that github will not complain about styling here.

Reply:
Added extra line